core-web/libs/portlets/dot-experiments/portlet/src/lib/dot-experiments-configuration/components/dot-experiments-configuration-variants/dot-experiments-configuration-variants.component.ts

```diff
@@ -50,8 +44,6 @@ import { DotExperimentsConfigurationVariantsAddComponent } from '../dot-experime
         CommonModule,
         DotMessagePipeModule,
         DotIconModule,
-        UiDotIconButtonModule,
-        UiDotIconButtonTooltipModule,
         DotExperimentsConfigurationVariantsAddComponent,
```

Comment:
All  `p-button-link` to `p-button-text` except in Actions is `p-button-outlined`